Transfer/Sources/CreateTransferPresenter.swift

```diff
@@ -339,6 +331,7 @@ final class CreateTransferPresenter {
 
             case .success(let transfer):
                 strongSelf.availableBalance = transfer?.destinationAmount
+                if strongSelf.didTapTransferAllFunds { strongSelf.amount = strongSelf.availableBalance ?? "" }
```

Comment:
refer code `var didTapTransferAllFunds: Bool = false {
        didSet {
            if didTapTransferAllFunds {
                amount = availableBalance ?? "0"
                view?.updateTransferAmountSection()
            }
        }
    }` so default value should be 0 instead of empty

Reply:
Yeah sure